### PR TITLE
Log level adjustments

### PR DESCRIPTION
**Pull request by lkemmerer**
I tried to break these up by changes between level (say info -> trace) in the attempt to make it easier to reason about each change. Not sure that's going to work, which is unfortunate because this is a big diff. :(

Tried to base the changes on the guidelines from here: https://confluence.flightglobal.io/display/DDT/Logging+Guidelines

### CODE REVIEW

File: src/main/java/com/flightstats/hub/app/ShutdownManager.java [hunk 1]
```diff
@@ -128,7 +128,7 @@ protected void startUp() throws Exception {
                     resetLock();
                 }
             } catch (KeeperException.NoNodeException e) {
-                log.info("node not found for ..." + PATH);
+                log.warn("node not found for ...", PATH);
```
Comment (lucasmo) [inline src/main/java/com/flightstats/hub/app/ShutdownManager.java]: I think this is missing a {}

File: src/main/java/com/flightstats/hub/webhook/LocalWebhookManager.java [hunk 2]
```diff
@@ -54,7 +54,7 @@ public void runAndWait(String name, Collection<String> keys, Consumer<String> co
         pool.shutdown();
         try {
             final boolean awaitTermination = pool.awaitTermination(5, TimeUnit.MINUTES);
-            log.info("awaitTermination", awaitTermination);
+            log.debug("awaitTermination", awaitTermination);
```
Comment (Paul-Hess) [inline src/main/java/com/flightstats/hub/webhook/LocalWebhookManager.java]: {}

File: src/main/java/com/flightstats/hub/webhook/WebhookService.java [hunk 3]
```diff
@@ -150,7 +150,7 @@ WebhookStatus getStatus(Webhook webhook) {
     }
 
     public void delete(String name) {
-        log.info("deleting webhook " + name);
+        log.debug("deleting webhook " + name);
```
Comment (Paul-Hess) [inline src/main/java/com/flightstats/hub/webhook/WebhookService.java]: {}

File: src/main/java/com/flightstats/hub/webhook/WebhookService.java [hunk 4]
```diff
@@ -166,7 +166,7 @@ private void deleteInstancesIfTagWebhook(String webhookName) {
         if (!webhookOptional.isPresent()) return;
         final Webhook webhook = webhookOptional.get();
         if (!webhook.isTagPrototype()) return;
-        log.info("TagWebHook: Deleting tag webhook instances for tag " + webhookName);
+        log.debug("TagWebHook: Deleting tag webhook instances for tag " + webhookName);
```
Comment (Paul-Hess) [inline src/main/java/com/flightstats/hub/webhook/WebhookService.java]: {}

File: src/main/java/com/flightstats/hub/webhook/error/WebhookErrorRepository.java [hunk 5]
```diff
@@ -33,7 +33,7 @@ public void add(String webhook, String error) {
     }
 
     public void deleteWebhook(String webhook) {
-        log.info("deleting webhook errors for " + webhook);
+        log.debug("deleting webhook errors for " + webhook);
```
Comment (Paul-Hess) [inline src/main/java/com/flightstats/hub/webhook/error/WebhookErrorRepository.java]: {}

File: src/main/java/com/flightstats/hub/app/ShutdownManager.java [hunk 6]
```diff
@@ -110,7 +110,7 @@ private void waitForLock() throws Exception {
                     curatorFramework.create().forPath(PATH, hostAddress.getBytes());
                     return;
                 } catch (Exception e1) {
-                    log.info("why did this fail?", e1);
+                    log.warn("why did this fail?", e1);
```
Comment (Paul-Hess) [inline src/main/java/com/flightstats/hub/app/ShutdownManager.java]: {}

File: src/main/java/com/flightstats/hub/spoke/ChannelTtlEnforcer.java [hunk 7]
```diff
@@ -60,7 +60,7 @@ protected void runOneIteration() throws Exception {
                 ttlEnforcer.deleteFilteredPaths(spokePath, channelService, handleCleanup());
                 log.info("completed channel cleanup {}", (System.currentTimeMillis() - start));
             } catch (Exception e) {
-                log.info("issue cleaning up channels in spoke", e);
+                log.error("issue cleaning up channels in spoke", e);
```
Comment (Paul-Hess) [inline src/main/java/com/flightstats/hub/spoke/ChannelTtlEnforcer.java]: {}

File: src/main/java/com/flightstats/hub/spoke/SpokeTtlEnforcer.java [hunk 8]
```diff
@@ -99,7 +99,7 @@ void cleanup() {
             statsdReporter.gauge(buildMetricName("ttl", "enforcer", "runtime"), runtime);
 
         } catch (Exception e) {
-            log.info("issue cleaning up spoke", e);
+            log.error("issue cleaning up spoke", e);
```
Comment (Paul-Hess) [inline src/main/java/com/flightstats/hub/spoke/SpokeTtlEnforcer.java]: {}

File: src/systemTest/java/com/flightstats/hub/system/service/S3Service.java [hunk 9]
```diff
@@ -139,7 +139,7 @@ public boolean confirmItemNotInS3(ChannelType storage, String fullPath, String c
         try {
             return !confirmItemInS3(storage, fullPath, channelName);
         } catch (AmazonS3Exception e) {
-            log.info("unexpected error: ", e.getMessage());
+            log.warn("unexpected error: ", e.getMessage());
```
Comment (Paul-Hess) [inline src/systemTest/java/com/flightstats/hub/system/service/S3Service.java]: {}

**Comment by lkemmerer** [comment]
Most of the instances that look like they're missing an interpolation marker use a Throwable as the last argument. Slf4j figures that out and prints out the stacktrace without us having to add the marker. See https://www.slf4j.org/faq.html#paramException
I think there are one or two instances where I either need to make sure it's a throwable and not just the .getMessage() or add a {}. I'll review and fix them.

**Comment by Paul-Hess** [comment]
Oh my bad, I didn't know about that feature, cool, thanks.

**Pull request merged**